core/services/gateway/handlers/functions/subscriptions.go

```diff
@@ -203,3 +225,32 @@ func (s *onchainSubscriptions) getSubscriptionsCount(ctx context.Context, blockN
 		Context:     ctx,
 	})
 }
+func (s *onchainSubscriptions) loadCachedSubscriptions() {
+	offset := uint(0)
+	for {
+		csBatch, err := s.orm.GetSubscriptions(offset, s.config.CacheBatchSize)
+		if err != nil {
+			break
+		}
+
+		for _, cs := range csBatch {
+			// only load subscriptions from cache db that are currently not present.
+			// if the subscription data is already present its more up to date.
```

Comment:
Is this comment still true? You are loading from DB only once on startup so the in-memory cache is empty.

Reply:
true, removing it.